app/presenters/workshop_presenter.rb

```diff
@@ -43,6 +64,16 @@ def time
     I18n.l(model.time, format: :time)
   end
 
+  def end_time
+    I18n.l(model.ends_at, format: :time)
+  end
+
+  def start_and_end_time
+    start_and_end_time = time
```

Comment:
@despo can we rename the `time` method to `start_time` to make it a bit more explicit what is happening here?

Reply:
After having a better look at this I'm not sure I want to make this change as part of this PR because
- `time` is used across the Event and WorkshopPresenter as it's the expected field in other event related models as well (Workshop, Event and Meeting to be precise), so to change it here I need to make a lot of additional changes to support this
- `time` should first be renamed in the DateTimeConcerns module, but as that is used across other models as well the work to verify that is behaves correctly everywhere is a lot
- the method is already used around a number of places app and also in shared event  related partials
- time is derived from the Workshop model's `date_and_time` so it would probably make sense to tackle from this level and rename the field to be `start_date_and_time`

As there are way too many changes to be made to resolve this are you happy with me opening an issue so this can be tackled separately?

Reply:
@despo I'm happy with you opening an issue for this to be tackled separately. Should have brought this up when `end_time` was introduced, but only noticed it here.

Reply:
@matyikriszta  New issue: https://github.com/codebar/planner/issues/1224